images/win/scripts/Installers/Install-DotnetSDK.ps1

```diff
@@ -107,6 +107,9 @@ function RunPostInstallationSteps()
 
     # Update Run key to run a script at logon
     Set-ItemProperty -Path "HKLM:\Software\Microsoft\Windows\CurrentVersion\Run" -Name "DOTNETUSERPATH" -Value $cmdDotNet
+
+    # Clear list of local NuGet resources in temporary cache
+    Invoke-Expression "dotnet nuget locals all --clear"
```

Comment:
nitpick: Why should we use Invoke-Expression? -> dotnet nuget locals all --clear

Reply:
Done